text/0079-cdk-v2.md

```diff
@@ -0,0 +1,339 @@
+---
+feature name: cdk-v2
+start date: 2020-28-01
+rfc pr: (leave this empty)
+related issue: #79
+---
+
+# Summary
+
+[Version 1.0 of the AWS CDK](https://github.com/aws/aws-cdk/releases/tag/v1.0.0)
+have been released about 6 months ago (July 2019) after about a year in
+developer preview. Since then, we have released over 25 minor versions (and
+about 10 patches) which included hundreds of bugs fixes and features.
+
+Throughout this time, we were able to deliver changes to the CDK without
+introducing (intentional) breaking changes to APIs marked as STABLE (when an
+unintentional break was introduced, we quickly released a patch to revert it).
+In a few cases, we introduced breaking changes to EXPERIMENTAL modules.
+
+This RFC proposes content and explores the implications of releasing a new major
+version for the AWS CDK (v2.0).
+
+# Motivation
```

Comment:
We should include a section in motivation with all the cleanup activities we take advantage of with a new major version. You have a good list here in the description - https://github.com/aws/aws-cdk-rfcs/issues/79.

And I'll add one to the list 'reserved word warnings' - https://github.com/aws/jsii/commit/5389def583160d6e06bb399704ff4cce9c8ef13a

Reply:
Added a link to #79, although I am not sure about all these changes. Might be too much.